src/Components/Footer/Footer.jsx

```diff
@@ -1,22 +1,29 @@
-import React from 'react'
+import React, { Component } from 'react'
 import Colour from '../../images/ColourWhite.png'
 
 import './Footer.scss'
 
 
-function  Footer() {
-  return (
-    <div className='footer'>
-      <img src={Colour} className='logo-white' />
-        <ul className='footer-link-list'>
-          <li><a href='#'>HOME</a></li>
-          <li><a href='#'>ABOUT</a></li>
-          <li><a href='#'>BLOG</a></li>
-          <li><a href='#'>PRIVACY POLICY</a></li>
-          <li><a href='#'>TERMS & CONDITIONS</a></li>
-        </ul>
-    </div>
-  );
+class Footer extends Component {
+  render() {
+    let home = this.props.page === 'HOME' ? 'active' :'inactive'
```

Comment:
Use `const { page } = this.props` and after that you don't need  `this.props.page` just `page`